src/SFA.DAS.EmployerAccounts.UnitTests/App.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+  <appSettings>
+    <add key="EnvironmentName" value="LOCAL"/>    
```

Comment:
could set this in whichever tests require it rather than adding the config: `ConfigurationManager.AppSettings["EnvironmentName"] = "LOCAL";`

This way could test behaviour when = "NOT_LOCAL"

Reply:
Done 👍